.gitignore

```diff
@@ -138,3 +138,6 @@ dmypy.json
 
 # Cython debug symbols
 cython_debug/
+
+# JetBrains
+.idea/
```

Comment:
I like to add stuff like this to my [global .gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c)

Reply:
Fair enough. If you ask GitHub to create a Python .gitignore for you, it gives you this block:

```
# PyCharm
#  JetBrains specific template is maintained in a separate JetBrains.gitignore that can
#  be found at https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore
#  and can be added to the global gitignore or merged into this file.  For a more nuclear
#  option (not recommended) you can uncomment the following to ignore the entire idea folder.
# .idea/
```

I usually take the nuclear option when I'm working on something that I expect will be touched by students. ;)